networkx/algorithms/community/asyn_lpa.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+#    Copyright (C) 2015
+#    All rights reserved.
+#    BSD license.
+
+import random
+import networkx as nx
+from itertools import groupby
+from collections import Counter
+
+
+def asyn_lpa_communities(G, weight=None):
+    """
+    Asynchronous label propagation algorithm for community detection described
+    on [1]_. The algorithm is non-deterministic and the found communities may
```

Comment:
The term "non-deterministic" is ambiguous; "randomized" or "probabilistic" is better.